docs/testcases/UI/Data_storages/2233/2233_1.md

```diff
@@ -0,0 +1,37 @@
+# Check webdav access duration preference
+
+Test verifies that 
+- time period for the datastorage to be mounted to dav can be defined
+- after this time storage should be automatically unmounted from WebDav
+
+**Prerequisites**:
+- Admin user
+- User group
+- Non-admin user is a member of this user group
+
+**Preparations**
+1. Login as admin user from the prerequisites
+2. Create the object storage
+3. Give *READ* permissions on that storage to the user group from the prerequisites
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the **Settings** page | |
+| 3 | Click the **PREFERENCES** tab | |
+| 4 | Click the **Data Storage** tab | |
+| 5 | Find the **`storage.webdav.access.duration.seconds`** preference |  |
+| 6 | Save the preference value, then clear and input `20` | |
```

Comment:
Add text like like that: "Enable the eye-icon near the preference if it is disabled"

Reply:
Done